jiant/tasks/tasks.py

```diff
@@ -2616,19 +2632,23 @@ def _load_split(data_file):
         )
         log.info("\tFinished loading COPA (as QA) data.")
 
-    def process_split(self, split, indexers) -> Iterable[Type[Instance]]:
+    def process_split(self, split, indexers, boundary_token_fn) -> Iterable[Type[Instance]]:
         """ Process split text into a list of AlleNNLP Instances. """
-        is_using_bert = "bert_wpm_pretokenized" in indexers
+        is_using_pytorch_transformers = "pytorch_transformers_wpm_pretokenized" in indexers
 
         def _make_instance(context, choices, question, label, idx):
             d = {}
-            d["question_str"] = MetadataField(" ".join(context[1:-1]))
-            if not is_using_bert:
-                d["question"] = sentence_to_text_field(context, indexers)
+            d["question_str"] = MetadataField(" ".join(context))
+            if not is_using_pytorch_transformers:
+                d["question"] = sentence_to_text_field(boundary_token_fn(context), indexers)
             for choice_idx, choice in enumerate(choices):
-                inp = context + question[1:] + choice[1:] if is_using_bert else choice
+                inp = (
+                    boundary_token_fn(context + question, choice)
```

Comment:
The old code seems to do `[CLS] context [SEP] question [SEP] choice [SEP]` but I think you can probably get away with any of these (this is consistent with what they did in the CommonsenseQA paper).

Reply:
I'll switch to `context [sep] question choice` to be consistent with the above. Using more than two `[sep]` tokens makes me nervous unless there's a clear need to.